README.md

```diff
@@ -13,6 +13,18 @@ Control plane clusters alert rules are maintained by the same team, but are kept
 different
 [repository](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/resources/stonesoup/argocd-control-plane/monitoring).
 
+The different alerting rules in this repository are:
```

Comment:
What's the reason to have the links pointing to a specific commit? It means that if an SOP changes, we'll need to update those links.

Reply:
@yftacherzog  I didn't notice it. I have updated it to the main branch now.